src/pages/commits/commitChart/Grid.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+
+const { gray } = uiColors;
+
+export const Grid: React.FC = () => (
+  <ColumnContainer>
+    <DashedLine />
```

Comment:
Can we maybe have a separated component that we can pass a number to, for example 5, and have it render that many lines?